### PR TITLE
Assignment-2

### PR DESCRIPTION
**Pull request by meta-pulkit-saxena**

### CODE REVIEW

File: CaseStudy/Assignment-2/src/resources/Vehicle.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  		<engineInCC>1200</engineInCC> 
+  		<fuelCapacity>30</fuelCapacity> 
+  		<mileage>21</mileage> 
+  		<price>550000</price> 
+  		<roadTax>25000</roadTax> 
+  		<AC>true</AC> 
+  		<powerSteering>true</powerSteering> 
+  		<accessoryKit>true</accessoryKit> 
+  	</Car>
+  		<Bike>
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/Vehicle.xml]: Remove tab space to sync format.

File: CaseStudy/Assignment-2/src/resources/09_21_2015.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+-- Creating the vehicle table.
+
+CREATE TABLE IF NOT EXISTS vehicle (
+    id INT,
+    created_by VARCHAR(20),
+    created_time DATE(25),
+    make VARCHAR(20),
+    model VARCHAR(20),
+    engine_In_CC INT,
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/09_21_2015.sql:17]: name of column should be in sync i.e in lowercase.

File: CaseStudy/Assignment-2/src/resources/09_21_2015.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    PRIMARY KEY(id)
+    );
+    
+-- Creating the car table.
+
+CREATE TABLE IF NOT EXISTS car (
+    id INT,
+    ac ENUM ("true", "false") NOT NULL DEFAULT "false",
+    power_steering ENUM ("true", "false") NOT NULL DEFAULT "false",
+    accessoryKit ENUM ("true", "false" ) NOT NULL DEFAULT "false",
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/09_21_2015.sql:31]: name of column should be in sync i.e in lowercase.

File: CaseStudy/Assignment-2/src/resources/09_21_2015.sql [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+CREATE DATABASE IF NOT EXISTS vehicle_database;
+
+-- Enabling the use of the vehicle_database.
+
+USE vehicle_database;
+
+-- Creating the vehicle table.
+
+CREATE TABLE IF NOT EXISTS vehicle (
+    id INT,
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/09_21_2015.sql:12]: here id can be used as auto generated.

File: CaseStudy/Assignment-2/src/resources/09_21_2015.sql [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    price DOUBLE(10,2),
+    road_Tax DOUBLE(10,2),
+    PRIMARY KEY(id)
+    );
+    
+-- Creating the car table.
+
+CREATE TABLE IF NOT EXISTS car (
+    id INT,
+    ac ENUM ("true", "false") NOT NULL DEFAULT "false",
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/09_21_2015.sql:29]: Create a separate column vehicle_id to be used foreign key from vehicle table.

File: CaseStudy/Assignment-2/src/resources/09_21_2015.sql [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    accessoryKit ENUM ("true", "false" ) NOT NULL DEFAULT "false",
+    PRIMARY KEY (id),
+    FOREIGN KEY ( id ) REFERENCES vehicle(id) ON DELETE CASCADE
+);
+
+-- Creating the bike table.
+
+CREATE TABLE IF NOT EXISTS bike (
+    id INT,
+    self_start ENUM ("true", "false") NOT NULL DEFAULT "false",
```
Comment (shikhar-metacube) [inline CaseStudy/Assignment-2/src/resources/09_21_2015.sql:40]: Create a separate column vehicle_id to be used foreign key from vehicle table.

File: CaseStudy/Assignment-2/src/resources/Vehicle.xml (hunk 1 above)
Comment (meta-pulkit-saxena) [inline CaseStudy/Assignment-2/src/resources/Vehicle.xml]: Done

**Pull request merged**